src/components/tables/Table.tsx

```diff
@@ -161,12 +159,47 @@ export class Table extends React.Component<ITableProps, {}> {
         <table className={tableClasses}>
           <TableChildLoadingRow {...this.props} isInitialLoad={this.isInitialLoad} />
           <TableChildHeader {...this.props} />
-          {TableChildBody(this.props)}
+          {this.getTableBody()}
         </table>
         <TableChildBlankSlate {...this.props} isInitialLoad={this.isInitialLoad} />
         <TableChildNavigation {...this.props} />
         <TableChildLastUpdated {...this.props} />
       </div>
     );
   }
+
```

Comment:
the following two methods should go above render https://github.com/coveo/tsjs/tree/master/react_redux#ordering

Reply:
Didn't we say that private functions should come after public ones?

Reply:
i guess these two rules are clashing with each other than...we could consider tossing out the `private` concept out of react-vapor...anyways it's not major you can leave it like this